tutorials/calibration_flow/gptq_like.py

```diff
@@ -0,0 +1,273 @@
+"""
+This is a example flow for GPTQ like calibration flows, where we:
+(1) optimize (quantize) one module at a time
+(2) with each optimization step, we need to get a set of all calibration data
+(3) the output of each module is calculated based on the optimized (quantized) module, and then pass down to the next module
+
+In this tutorial we mainly use two things:
+(1) MultiTensor subclass https://gist.github.com/HDCharles/a1b575bbf8875f994af8a01b225e1227
```

Comment:
n00b q but it's not super clear to me what is a multi tensor exactly and why it's relevant here?

Reply:
MultiTensor is a Tensor that collects a list of Tensors (calibration data) and it can be used for optimizing a single module, and also the module can output the MultiTensor result for output. maybe I can add some motivation here